en/docs/learn/design-api/advanced-topics/adding-an-api-state-change-workflow.md

```diff
@@ -62,8 +62,10 @@ First, enable the API state change workflow for **Approval Workflow Executor**.
 
 The elements of the above configuration are explained below.
 
-| Element name                                       | Description                                                                                                                                               |
-|----------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|                                                                f                                                                  |
-| `stateList`| Mandatory          | This is a comma separated list of the current state and intended action. For example, Created:Publish,Published:Block 
-
-                                                                |
+| Element Name               | Mandatory/Optional | Description                                                                                                                                                                                         |
```

Comment:
we should point to this from the 3rd point where the configuration is done.

Reply:
Done with 20c4d8a
Added missing link